stock_cycle_count/models/stock_cycle_count.py

```diff
@@ -114,7 +117,11 @@ def action_create_inventory_adjustment(self):
                 inv.prefill_counted_quantity = (
                     self.company_id.inventory_adjustment_counted_quantities
                 )
-                inv.action_state_to_in_progress()
+                try:
+                    inv.action_state_to_in_progress()
+                except Exception as e:
```

Comment:
Isn't it a too broad Exception ? 

Moreover, shouldn't we manage the exception in a more elegant manner than 'just' logging ?

Reply:
Hey @rousseldenis 

You might be right. I implemented it this way because we don't want the cron job to stop running due to a validation error from a location that already has an inventory adjustment in progress.

Do you have any suggestions for a better approach?

Thanks.